src/runtime/auto-prompt-manager.js

```diff
@@ -187,6 +189,15 @@ export class AutoPromptManager {
     dismissedPrompts,
     params
   ) {
+    const audienceActionsPromptType = article
+      ? this.getAutoPromptType_(article)
```

Comment:
Could you pass the `article.audienceActions` directly here?

Reply:
Seeing https://github.com/subscriptions-project/swg-js/blob/main/src/runtime/entitlements-manager.js#L53, the audienceActions isn't really a defined type, so in this file, it's usually the "article" value that gets passed around. If we were to pass audienceActions, what should I put here as the param type? https://github.com/subscriptions-project/swg-js/blob/3c29a8084d4ce843d2ff887fd89eb39a15da0f84/src/runtime/auto-prompt-manager.js#L429

Reply:
Oh, good call. Could you pass `article.audienceActions?.actions` as a `string[]`?